index.html

```diff
@@ -55,7 +55,31 @@
          </div>
       </header>
       <!--Main-->
-      <main></main>
+      <main>
+         <!--Partners section-->
+         <div class="partners">
+         <div id="partners-head">
```

Comment:
I think that for what we are doing now is better if you use a class instead of an id ... just because we are not gonna use JS.

Reply:
I changed it :)